Cargo.toml

```diff
@@ -21,7 +21,8 @@ prost = "0.11"
 prost-types = "0.11"
 serde = "1.0"
 snafu = "0.8"
-stackable-operator = { git = "https://github.com/stackabletech/operator-rs.git", tag = "stackable-operator-0.67.0" }
+# I did not want to release op-rs 0.68.0 just for this change...
+stackable-operator = { git = "https://github.com/stackabletech/operator-rs.git", rev = "e93736000b4bcb3027550d2e0a52fafa345c050c" }
```

Comment:
Should probably be fixed before merging.

Reply:
I have to admit I would be happy to merge as is but it's not a strong opinion (aka what's the benefit of pinning a tag over pinning a commit?).
If you want I will create an op-rs release and use that here